### DISCUSSION TITLE
Connecting to wallet ....

### DISCUSSION THREAD

**Discussion by wc357** [Chia-Network/chia-blockchain#8819] [Support]
I tried to wait three hours and found that the Chia program had crashed.I tried this several times.

Rumor has it that Chia's IP and mine pool IP have been banned by the Chinese government.So the pool netspace of Asia has decreased over the last two days???

win10
10.13

**Comment by BrandtH22** [comment]
@wc357 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
